docs/excel/custom-functions-errors.md

```diff
@@ -58,12 +57,50 @@ let error = new CustomFunctions.Error(CustomFunctions.ErrorCode.invalidValue, "T
 throw error;
 ```
 
-## Use try-catch blocks
+### Output an array with an error
```

Comment:
Does this apply when your custom function returns dynamic arrays? If so might be more clear to title this as "Handle an error when working with dynamic arrays" or similar.